app/views/admin/pages/show.html.erb

```diff
@@ -1,11 +1,11 @@
 <form id="slices-form">
   <div id="container-actions">
-    <% if Slices::Translations.available? %>
+    <% if I18n.available_locales.many? %>
       <div class="toolbar translations">
         <nav>
           Edit in
           <ul>
-            <% Slices::Translations.all.each do |locale, language| %>
+            <% I18n.available_locales.each do |locale, language| %>
```

Comment:
Ok I can't find an easy way. How about we do:

```ruby
language = I18n.t("locales.#{locale}", default: locale)
```

So look in the translations to see if `locales.en` or `locales.de` exists, and if not just fall back to "en" or "de". This has the added benefit of allowing translation of the locale names. I.e. right now if the admin UI was translated into German, there would be no way to make the buttons say "Englisch" and "Deutsch"!